tests/run.yml

```diff
@@ -0,0 +1,52 @@
+---
+#Run main playbook
+- hosts: streisand
+  gather_facts: no
+  remote_user: root
+  become: yes
+  pre_tasks:
+    - name:
+      raw: sudo apt update && apt install python -y
+
+- hosts: streisand
+  gather_facts: yes
+  remote_user: root
+  become: yes
+  tasks:
+    - name: Get the default SSH key
```

Comment:
I'm slightly nervous about having a SSH private key checked into the repo that's used by default for the `tests/run.yml` setup. One day someone might end up doing something silly and leaving a container running that's internet exposed accessible with a published SSH private key.

Would it be possible to generate a unique SSH key per container & use that? Is there a reason you preferred this approach I'm overlooking?